tests/runtest.sh

```diff
@@ -1028,6 +1028,7 @@ if ((disableEventLogging == 0)); then
     export COMPlus_EnableEventLog=1
 fi
 
+export COMPlus_JitDump=AppendFormatHelper
```

Comment:
FWIW, specifying the name this way doesn't work for me locally. `export COMPlus_JitDump=*AppendFormatHelper` does work though.

Also, I'm not sure how much you care about this, but depending on what happens at execution time, the output from the JIT dump will probably be interleaved with other messages in the console, making it hard to read. If that's the case, you may want to write it to a file instead of printing it to the console, and then just print the file at the end after everything is done running.